src/context/cart-context.js

```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { createContext, useContext } from "react";
+import { useState ,useEffect } from "react";
+import { useAuth } from "./auth-context";
+
+
+const cartContext = createContext(null);
+const useCart = () => useContext(cartContext);
+
+const authtoken = localStorage.getItem("AuthToken")  
```

Comment:
You should move this inside the provider.

Reply:
yes, fixed! thanks😀